worlds/witness/player_logic.py

```diff
@@ -322,7 +322,9 @@ def make_options_adjustments(self, world: "WitnessWorld"):
         elif victory == 3:
             self.VICTORY_LOCATION = "0xFFF00"
 
-        if chal_lasers <= 7:
+        # Long box can usually only be solved by opening the mountain entry. If it's 7 lasers or less, that's no longer
+        # true. Also, if the user sets mountain_lasers > 7, the box is rotated to not require the hatch either.
```

Comment:
The first point refer to the common "long box" name while the second refer to the option name. Could they be changed in a more uniform way (both as options name or both as "common" names)?

Reply:
The physical panel solution is called "long box". The amount of lasers you need for it & the corresponding setting controlling that amount is called "challenge lasers"

Reply:
I.e., this is an example of a correctly phrased sentence: "The amount of lasers needed to solve long box is controller by the challenge_lasers setting"

Reply:
I could phrase the sentence a bit more clearly though, the use of "it" is somewhat imprecise